src/components/AdminSettings.vue

```diff
@@ -71,11 +71,21 @@
 				<div>
 					<input id="demoserver" v-model="serverMode" type="radio"
 						name="serverMode" value="demo" class="radio"
-						:disabled="updating" @input="fetchDemoServers">
+						:disabled="updating || hasHostErrors" @input="fetchDemoServers">
 					<label for="demoserver">{{ t('richdocuments', 'Use a demo server') }}</label><br>
 					<p class="option-inline">
 						<em>{{ t('richdocuments', 'You can use a demo server provided by Collabora and other service providers for giving Collabora Online a try.') }}</em>
 					</p>
+					<div v-if="hasHostErrors" class="option-inline-emphasized">
+						<p>{{ t('richdocuments', 'Your NextCloud setup is not capable of connecting to the demo servers because;') }}</p>
+						<ul>
+							<li v-if="hostErrors[0]">{{ t('richdocuments', 'it is a local setup (localhost)') }}</li>
```

Comment:
The linter complains about the html single line formatting. Running `npm run lint:fix` should be able to fix that automatically :wink:

Reply:
I had tried `npm run dev --fix`, but seems like this is the right command. Thanks! :)